### ISSUE TITLE
Request for New Lighthouse Audit: Vector Image Optimization

### ISSUE THREAD

**Issue by LarsFlieger** [GoogleChrome/lighthouse#16285]
**Feature request summary**

I propose adding a new audit to Lighthouse that identifies images that could potentially be vector graphics and estimates the potential savings in file size and performance if these images were converted to vector formats.

**What is the motivation or use case for changing this?**
Many websites use pixel-based images for elements that are inherently vector in nature, such as logos, icons, and simple illustrations. Converting these to vector formats like SVG can significantly reduce file sizes and improve scalability, leading to better performance and user experience.

**How is this beneficial to Lighthouse?**

- Reduced page load times and bandwidth usage
- Improved scalability of images across different device sizes
- Better overall performance scores in Lighthouse

**Comment by adamraine** [comment]
Using vector images is something devs should always consider, but outside of some advanced image analysis I don't see how we can recommend this on a case-by-case basis. I don't think we can do more than just mention this as a possibly strategy in our docs, [which we already do](https://developer.chrome.com/docs/lighthouse/performance/uses-responsive-images#:~:text=Another%20strategy%20is%20to%20use%20vector%2Dbased%20image%20formats%2C%20like%20SVG.%20With%20a%20finite%20amount%20of%20code%2C%20an%20SVG%20image%20can%20scale%20to%20any%20size.%20See%20Replace%20complex%20icons%20with%20SVG%20to%20learn%20more.).

**Comment by LarsFlieger** [comment]
> Using vector images is something devs should always consider, but outside of some advanced image analysis I don't see how we can recommend this on a case-by-case basis. I don't think we can do more than just mention this as a possibly strategy in our docs, [which we already do](https://developer.chrome.com/docs/lighthouse/performance/uses-responsive-images#:~:text=Another%20strategy%20is%20to%20use%20vector%2Dbased%20image%20formats%2C%20like%20SVG.%20With%20a%20finite%20amount%20of%20code%2C%20an%20SVG%20image%20can%20scale%20to%20any%20size.%20See%20Replace%20complex%20icons%20with%20SVG%20to%20learn%20more.).

Yes, checking if an pixel image is an vector based image is a difficult task, and I’m also unsure which method or algorithm could solve this problem efficiently enough to be integrated into Lighthouse. However, I think this feature should be considered with further investigation because its benefits could be significant.

Perhaps this should be implemented on a different platform, where integrating some form of AI to solve this problem might also be feasible.

**Comment by connorjclark** [comment]
Thanks for the suggestion, but we feel this is out of scope for Lighthouse. It would be an interesting tool though!
